article/app/views/fragments/mainMedia.scala.html

```diff
@@ -26,10 +26,11 @@
                                 showControlsAtStart = true,
                                 endSlatePath = EndSlateComponents.fromContent(article.content).toUriPath,
                                 overrideIsRatioHd = None,
-                                embedPath = article.content.mainVideoCanonicalPath
+                                embedPath = Some(article.content.mainVideoCanonicalPath),
```

Comment:
`article.content.mainVideoCanonicalPath` should never be `None` as it comes from CAPI.